FrameworkSystem/private/standardLogging/gLoggingRoot.py

```diff
@@ -0,0 +1,132 @@
+"""
+gLogging Root
+"""
+
+__RCSID__ = "$Id$"
+
+import logging
+import time
+import sys
+
+from DIRAC.FrameworkSystem.private.standardLogging.LogLevels import LogLevels
+from DIRAC.FrameworkSystem.private.standardLogging.gLogging import gLogging
+
+
+class gLoggingRoot(gLogging):
+  """
+  gLoggingRoot is a gLogging object and it is particular because it is the first parent of the chaine.
+  In this context, it has more possibilities because it is the one that initializes the logger of the 
+  standard logging library and it configures it with the cfg file.
+
+  There is a difference between the parent gLogging and the other because the parent defines the behaviour
+  of all the gLogging objects, so it needs a specific class.  
+
+  gLoggingRoot has to be unique, because we want one and only one parent on the top of the chain: that is why 
+  we created a singleton to keep it unique. 
+  """
+
+  __configuredLogging = False
```

Comment:
It prevents the multiple configuration of the gLoggingRoot object. Actually, I don't know if it is very useful, it was from the old gLogger.